vanilla-app/integrations/form.html

```diff
@@ -45,14 +45,18 @@ <h1>Vanilla - PIE Form Test Page</h1>
             <icon-key slot="leadingIcon"></icon-key>
         </pie-text-input>
 
+        <pie-form-label for="description">Description:</pie-form-label>
+        <pie-textarea class="form-field" id="description" data-test-id="description" name="description">
```

Comment:
Is there an error in the console about a missing `value`? Or is that only a typescript error?

Reply:
There isn't an error in console